partialWaveFit/fitResult.cc

```diff
@@ -998,3 +1000,151 @@ fitResult::intensityErr(const string& waveNamePattern) const
 	const TMatrixT<double> intensityCov = jacobian * prodAmpCovJT;              //  1 x  1 matrix
 	return (double)normNmbEvents() * sqrt(intensityCov[0][0]);
 }
+
+
+void fitResult::stripCovMatrix() {
+	_fitParCovMatrix.ResizeTo(0,0);
+	_fitParCovMatrixIndices.clear();
+	_hasHessian = false;
+	_covMatrixValid = false;
+}
+
+
+void fitResult::stripIntegralMatrices() {
+	_normIntegral.resizeTo(0,0);
+	_acceptedNormIntegral.resizeTo(0,0);
+	_phaseSpaceIntegral.clear();
+	_normIntIndexMap.clear();
+}
+
+
+std::map<rpwa::multibinBoundariesType, std::list<rpwa::fitResult> >
+rpwa::getFitResultsFromFilesInMultibins(
+                                        const std::vector<std::string>& fileNames,
+                                        const std::string& treeName,
+                                        const std::string& branchName,
+                                        const bool onlyBestInMultibin,
+                                        const bool stripMatricesFromFurtherAttempts,
+                                        const bool onlyConvergedResults) {
+	std::map<rpwa::multibinBoundariesType, std::list<rpwa::fitResult> > fitResultsInMultibins;
+	std::unique_ptr<fitResult> fitResultIn(new fitResult);
```

Comment:
The "In" is ambiguous. I guess it stands for input. If yes, then "inputFitResult" would be better.

Reply:
Yes, the "In" in fitResultsInMultibins means that the fit results are grouped into multibins.